### PR TITLE
initial check in for SCCM current branch

### PR DESCRIPTION
**Pull request by yizhongwu**
I messed up. so I recreated this repository and create new branch and check-in the tempalte again

### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: sccm-currentbranch/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+          }
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "apiVersion": "2018-06-01",
+      "name": "[concat(toLower(parameters('prefix')),variables('vmrole')[copyIndex(3)],'/WorkFlow')]",
+      "location": "[parameters('location')]",
+      "copy": {
```
[the remark this answers is not in the document]
Comment (yizhongwu) [reply sccm-currentbranch/azuredeploy.json]: Fixed. Putting all extension resources in one copy loop.

File: sccm-currentbranch/scripts/InstallSCCM.ps1 [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+		$url = "http://aka.ms/downloadazcopy"
+		Invoke-WebRequest -Uri $url -OutFile $path
+	}
+
+	#Install azcopy
+	Start-Process msiexec.exe -Wait -ArgumentList "/I $path /quiet"
+}
+
+"[$(Get-Date -format HH:mm:ss)] Copying SCCM installation source..." | Out-File -Append $logpath
+#SCCM installation file are not available from internet. We maintain it in our own storage currently.
```
[the remark this answers is not in the document]
Comment (bmoore-msft) [reply sccm-currentbranch/scripts/InstallSCCM.ps1]: Still need to resolve this one, but the rest of the changes look good.

[the remark this answers is not in the document]
Comment (yizhongwu) [reply sccm-currentbranch/scripts/InstallSCCM.ps1]: Now, the script is downloading from a public location. It's Microsoft download center.

**Comment by yizhongwu** [comment]
@bmoore-msft all fixes are done per your suggestion, please help review again.

**Pull request merged**